app-text/texlive/texlive-2022.recipe

```diff
@@ -195,7 +195,7 @@ GLOBAL_WRITABLE_FILES="
 	"
 
 PATCH() {
-	sed -i "s|\`kpsewhich -var-value=SELFAUTOPARENT\`|\"$dataDir/texlive\"|" $sourceDir/tlpkg/TeXLive/TLUtils.pm
+	sed -i "s|\`kpsewhich -var-value=TEXMFROOT\`|\"$dataDir/texlive\"|" $sourceDir/tlpkg/TeXLive/TLUtils.pm
```

Comment:
This is apparently what was meant with the change from SELFAUTOPARENT to TEXMFROOT, not the actual code in kpsewhich.